mesonbuild/compilers/d.py

```diff
@@ -27,6 +27,21 @@
     CompilerArgs,
 )
 
+d_feature_args = {
+                  'gcc':  {'unittest': '-funittest',
+                           'version': '-fversion',
+                           'import_dir': '-J'
+                           },
+                  'llvm': {'unittest': '-unittest',
+                           'version': '-d-version',
+                           'import_dir': '-J'
+                           },
+                  'dmd':  {'unittest': '-unittest',
+                           'version': '-version',
+                           'import_dir': '-J'
+                           }
+                 }
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->